torch/csrc/jit/script/sugared_value.h

```diff
@@ -350,6 +343,30 @@ static inline Self simpleSelf(const TypePtr& typ) {
   };
 }
 
+// Represents nested class namespaces, like `foo.bar.Baz`.
```

Comment:
Can you put this with the other import-specific sugared values in import_source.cpp? Otherwise people will get confused and try to use there in other places.